dist/src/main/config/broker.standalone.yaml.template

```diff
@@ -324,9 +324,9 @@
       # useWindowed: true
 
       # The algorithm configures which algorithm to use for the backpressure.
-      # It should be one of vegas, aimd, fixedLimit, gradient, or gradient2.
+      # It should be one of vegas, aimd, fixed, gradient, or gradient2.
       # This setting can also be overridden using the environment variable ZEEBE_BROKER_BACKPRESSURE_ALGORITHM
-      # algorithm: "vegas"
+      # algorithm: "fixed"
```

Comment:
```suggestion
      # algorithm: "vegas"
```

Reply:
I justed checked, and the default is still vegas in this PR https://github.com/zeebe-io/zeebe/pull/5317/files#diff-deea4dc9f416a8f4f5dd435a177d8b84R16. 

Is it okay if I merge @deepthidevaki ?

Reply:
Yes. :+1: